src/SignatureDecipher.php

```diff
@@ -0,0 +1,166 @@
+<?php
+	// Created by StefansArya for contribute to PHP Youtube-Downloader
+	// Because we will processing 1MB of file I will not use RegExp for processing strings
+
+	namespace YoutubeDownloader;
+
+	class SignatureDecipher
+	{
+		public static function downloadPlayerScript($videoID){
+			$playerID = self::loadURL("https://www.youtube.com/watch?v=$videoID");
+			$playerID = explode("\/yts\/jsbin\/player-", $playerID);
+			if(count($playerID)<=1){
+				echo("Failed to retrieve player script for video id: $videoID");
+				return false;
+			}
+			$playerID = $playerID[1];
+			$playerURL = str_replace('\/', '/', explode('"', $playerID)[0]);
+			$playerID = explode('/', $playerURL)[0];
+	
+			if(!file_exists("playerscript")) mkdir("playerscript");
```

Comment:
Can you please create the empty file `playerscript/.gitkeep` so this if-statement and the `mkdir` can be removed.